IOSAccessAssessment/Services/ChangesetService.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ChangesetService.swift
+//  IOSAccessAssessment
+//
+//  Created by Mariana Piz on 11.11.2024.
+//
+
+import Foundation
+
+struct NodeData {
+    let latitude: Double
+    let longitude: Double
+}
+
+class ChangesetService {
+    
+    private enum Constants {
```

Comment:
Just a small point, but we can later move the constants to the global constants file, for better centralization, or at least in a common folder for constants.